src/AureliaDependenciesPlugin.ts

```diff
@@ -21,8 +22,22 @@ class AureliaDependency extends IncludeDependency {
   get [dependencyImports]() {
     return webpack.Dependency.EXPORTS_OBJECT_REFERENCED as any;
   }
+
+  serialize(context: any) {
```

Comment:
do we know the type of this context? if we do, I'd prefer that we have it here instead of any

Reply:
It's `ObjectSerializerContext `. Unfortunately, this type is only listed but not exported in the [webpack ts definition file](https://github.com/webpack/webpack/blob/main/types.d.ts) if I see this correctly.